storagenode/gracefulexit/chore.go

```diff
@@ -53,6 +53,9 @@ func (chore *Chore) Run(ctx context.Context) (err error) {
 
 	err = chore.Loop.Run(ctx, func(ctx context.Context) (err error) {
 		defer mon.Task()(&ctx)(&err)
+
+		mon.Counter("satellite.request").Inc(1)
```

Comment:
Maybe it would be better to have this metric be a different one so they can be distinguished from one another?
Also, this will only increment the counter one time for *all* graceful exits. In fact, it will even increment the counter if there are no graceful exits, at every interval. You probably want to add the metric in the loop below (line 73ish)

Reply:
Metrics were renamed. I discussed with the Data team and we'll need all graceful exits (even if there are no graceful exits). Also, moved the code to the proper position as you mentioned.

Reply:
> we'll need all graceful exits (even if there are no graceful exits)
If this is true, then the original position is where you want the metric. Or you need to have a different metric in that position. The metric inside the loop will be triggered for every satellite (if 3 satellites are exiting, it will increment the counter 3 times, but if 0 are exiting, it won't increment the counter at all).